m4_sessions/1708-inning/group_two.md

```diff
@@ -21,16 +21,26 @@ What do we want people attending the session to learn or walk away with at the e
 * Being more actively aware of usability
 
 ### Session Outline:
-Intro to UX/UI (10 minutes)
-- need to come up with what this looks like
-Design Thinking Video (10 minutes or less)
-- need to find one or two great videos
+_We'll use a keynote slide deck to guide the session, which will be shared out at the end of the session_
+
+Intro to UX/UI (~10 minutes)
+* Facilitators introduce themselves, go around the room and do introductions and answer what ux/ui means to you
+
+Design Thinking Video (~15 minutes)
+* Video 1: https://www.youtube.com/watch?v=GFffb2H-gK0 (8:41 // A video produced by the Florida Hospital Innovation Labs covering the 5 different stages of design thinking, which are _empathy_, _reframe_, _brainstorming_, _prototype_, _testing_)
```

Comment:
I don't fully understand the interrupting guy, but this video does give clarity of each stage.